RELEASE.txt

```diff
@@ -126,29 +126,39 @@ Example `version number`
       git commit -m 'Bump version'
       git push origin main
 
-----
-
-- Once the release is completed you should update the release docs:
-  - Edit ``doc/source/_static/docversions.js`` and commit
-  - On the release branch, build a clean version of the docs. In the
-    root directory, run ``pip install .``.
-  - In the ``doc/`` directory:
-    - Build using
-      ``make clean; make html; make gh-pages``.
-    - Check (since this a new feature) that binder links in gallery examples
-      point to the release branch, e.g. `0.16.x`.
-    - In the ``gh-pages/`` directory:
-      - Update the symlink to ``stable`` and commit.
-      - Upload the docs: ``git push origin gh-pages`` in ``doc/gh-pages``.
+- Update documentation on the web:
+  The documentation is kept in a separate repo: scikit-image/docs
+
+  - Wait for the CI service to deploy to GitHub Pages
+  - Sync your branch with the remote repo: ``git pull``.
+  - Update the docs
+
+      git switch gh-pages
+      cp -a dev ../.
+      git reset --hard <commit from last release>
+      git rm -rf dev
+      cp -a ../dev .
+      git add dev
+      cp -a ../dev <release name, e.g., 0.23.x>
+      git add <release name, e.g., 0.23.x>
+      rm stable
+      ln -s <release name, e.g., 0.23.x> stable
+      git add stable
+      git commit -m "Add <release name, e.g., 0.23.x>"
+      git push -f origin gh-pages # force push---be careful!
```

Comment:
Perhaps we could recommend creating a local backup branch with

```suggestion
      git branch -C gh-pages gh-pages-backup
      git push -f origin gh-pages # force push---be careful!
```
before force-pushing?

Reply:
What would the advantage of the backup branch be?

Reply:
The same advantage as the comment "be careful". In case something broke in the previous steps there would still be a backup. But you are right, it should probably be created before any modification to the local branch otherwise it's useless. :sweat_smile: It's just a suggestion though.

Reply:
(Sorry, saw your edit only after my reply)

Reply:
No worries. When I first saw your comment, I was confused and thought maybe I was missing something. Then realized it was just being done in the wrong spot.